packages/core/ios/RNEmbrace/EmbraceManager.m

```diff
@@ -119,6 +119,26 @@ @interface RCT_EXTERN_MODULE(EmbraceManager, NSObject)
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject)
 
+RCT_EXTERN_METHOD(logNetworkRequest:(NSString *)url
+                  httpMethod:(NSString *)httpMethod
+                  startInMillis:(double)startInMillis
+                  endInMillis:(double)endInMillis
+                  bytesSent:(double)bytesSent
+                  bytesReceived:(double)bytesReceived
+                  statusCode:(int)statusCode
```

Comment:
even if we're treating as an int I think this should remain a double due to how RN translates parameter types over the bridge, e.g. https://reactnative.dev/docs/native-modules-ios#argument-types

Reply:
I see, okay. I'll make the change